### DISCUSSION TITLE
Can't install nl_core_web_sm

### DISCUSSION THREAD

**Discussion by yossigijrath** [explosion/spaCy#6535] [Help: Coding & Implementations] [answered]
Upgrade spacy to 2.3.4 and run the command to download a model:

python -m spacy download nl_core_web_sm
results in an error:

✘ No compatible models found for v2.3.4 of spaCy
Your Environment

spaCy version: 2.3.4
Platform: macOS-11.0.1
Python version: 3.8.6

**Comment by adrianeboyd** [comment, marked as answer]
The model is called `nl_core_news_sm`, not `nl_core_web_sm`. See the models here: https://spacy.io/models/
